packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/DecreaseWorkingGroupLeadStake.tsx

```diff
@@ -0,0 +1,123 @@
+import { TextInlineMedium, TextMedium } from '@/common/components/typography'
+import { capitalizeFirstLetter } from '@/common/helpers'
+import { useNumberInput } from '@/common/hooks/useNumberInput'
+import { formatTokenValue } from '@/common/model/formatters'
+import { SelectMember } from '@/memberships/components/SelectMember'
+import { useMember } from '@/memberships/hooks/useMembership'
+import { SelectWorkingGroup } from '@/working-groups/components/SelectWorkingGroup'
+import { useWorkingGroup } from '@/working-groups/hooks/useWorkingGroup'
+
+export interface DecreaseWorkingGroupLeadStakeParameters {
+  stakingAmount?: BN
+  groupId?: string
+  workerId?: number
+}
+
+interface DecreaseWorkingGroupLeadStakeProps extends DecreaseWorkingGroupLeadStakeParameters {
+  setStakingAmount: (amount: BN) => void
+
+  setGroupId(groupId: string): void
+
+  setWorkerId(workerId?: number): void
+}
+
+export const DecreaseWorkingGroupLeadStake = ({
+  stakingAmount,
+  groupId,
+  setStakingAmount,
+  setGroupId,
+  setWorkerId,
+}: DecreaseWorkingGroupLeadStakeProps) => {
+  const [amount, setAmount] = useNumberInput(0, stakingAmount)
+  const setHalf = () => setAmount(stakingAmount ? stakingAmount.divn(2).toString() : '')
+  const setThird = () => setAmount(stakingAmount ? stakingAmount.divn(3).toString() : '')
+
+  const { group, isLoading: isGroupLoading } = useWorkingGroup({ name: groupId })
+  const { member: leader } = useMember(group?.leaderId)
+
+  useEffect(() => setStakingAmount(new BN(amount)), [amount])
+  useEffect(() => setWorkerId(group?.leaderWorker?.runtimeId), [groupId, group?.leaderWorker?.runtimeId])
+
+  return (
+    <RowGapBlock gap={24}>
+      <Row>
+        <RowGapBlock gap={8}>
+          <h4>Specific parameters</h4>
+          <TextMedium lighter>Decrease Working Group Lead Stake</TextMedium>
+        </RowGapBlock>
+      </Row>
+      <Row>
+        <RowGapBlock gap={20}>
+          <InputComponent
+            label="Working Group"
+            required
+            inputSize="l"
+            tooltipText="Please select an identifier for Working Group"
+          >
+            <SelectWorkingGroup selectedGroupId={groupId} onChange={(selected) => setGroupId(selected.id)} />
+          </InputComponent>
+          {(!groupId || isGroupLoading || (group && leader)) && (
+            <>
+              <InputComponent label="Working Group Leader" inputSize="l" disabled>
+                <SelectMember onChange={() => true} disabled selected={leader} />
+              </InputComponent>
+              {group && (
+                <Info
+                  content={
+                    <TextMedium>
+                      The actual stake for {capitalizeFirstLetter(group.name)} Working Group Lead is{' '}
+                      <TextInlineMedium bold>{formatTokenValue(group.leaderWorker?.stake)} JOY</TextInlineMedium>.
+                    </TextMedium>
+                  }
+                ></Info>
```

Comment:
```suggestion
            />
```

This might be a collapsed block as it has no children.

Reply:
I've refactored this `<Info/>` component